src/PHPSQLParser/processors/ReplaceProcessor.php

```diff
@@ -50,8 +50,8 @@
  */
 class ReplaceProcessor extends InsertProcessor {
 
-    public function process($tokenList) {
-        return parent::process($tokenList, 'REPLACE');
+    public function process($tokenList, $token_category = 'REPLACE') {
```

Comment:
might want to use camel case here, for consistency, ie `$tokenCategory`

Reply:
oh, nevermind, I see now that the parent function uses `$token_category`, so you are already consistent :) sorry for the noise